fastlane/Fastfile

```diff
@@ -0,0 +1,74 @@
+# This file contains the fastlane.tools configuration
```

Comment:
We don't need to upload the generated XCFramework to the private artifactory right mas?

Reply:
So basically we don't need to use Fastlane